%\newpage
\cvsection{\faBuildingO\space Research Experience}

\begin{cventries}

\cventry
{Research Assistant- Department of Microbiology \& Cell Science - Advisor: Marc G. Chevrette, PhD}
{University of Florida}
{Gainesville, FL}
{09/2022--present}
%{}
{
\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {\textbf{Advisor:} Jo Handelsman, PhD}
\item {Computational pipeline to delimitate quantitatively the boundaries for 7-Deazaguanine proteins (PreQ) BGCs to discover active compounds, such as antibiotics or anti-cancer agents. To be submitted for publishing in the future; waiting to acquire further data.}
\item {Compilation of negative data for ATP-PPi exchange essays to predict NRPS A-domain specificity by machine learning algorithm trained with 3D structures. Manuscript in preparation.}
\item {Evolutionary reconstruction of PreQ in the Actinomycetota Phylum, and their implications for natural products discovery. To be submitted for publishing in the future; waiting to acquire further data.}
\end{cvitems}
}
\vspace{0cm}

\cventry
{Assistant Curator for the Microbial Collection - Advisor: Jorge L. Fuentes, PhD}
{Industrial University of Santander}
{Bucaramanga, Colombia}
{01/2021--08/2021}
{}

\vspace{-0.7cm}
\cventry
{Research Assistant - Biology School - Advisor: Jorge L. Fuentes, PhD}
{}
{}
{06/2019--12/2021}
{
\begin{cvitems} % Description(s) of tasks/responsibilities
\item {Responsible for reactivation, culture, DNA extraction, electrophoresis, and PCR of indigenous bacterial strains (e.g., Actinobacteria, Gammaproteobacteria, and Proteobacteria) from different environments of Colombia preserved in the culture collection "Cepario LMMA-UIS" (https://doi.org/10.15472/uq6pal)}
\item {Evaluated radioresistant properties in \textit{S. marcescens} indigenous strains with different ultraviolet radiation B doses (UVB). To be submitted for publishing in the future; waiting to acquire further data from other researchers.}
%\item {Led the establishment of a protocol for bacterial viability against UVB using staining and fluorescence microscopy.}
%\item {Evaluated prodigiosin production in environmental isolates of \textit{S. marcescens} in different culture media and temperatures.}
%\item {Extracted prodigiosin pigment from 40 environmental isolates of \textit{S. marcescens} and tested in vitro photoprotection indices, genotoxic and cytotoxic activities as well as antigenotoxic efficacy against UVB using SOS Chromotest Essay.}
%\item {Determined the relationships between prodigiosin concentration, SPF values, GI estimates, and environmental traits (temperature, rainfall, irradiance solar, and altitude) of the isolation sites by Pearson correlation analysis and principal component analysis (PCA).}
\item {\textbf{Discovery}: prodigiosin yield depended on the culture media used for its growth and correlated with the in-situ temperature and irradiance solar traits. This natural product acts as a filter of UV radiation with broad-spectrum photoprotection (UVA-UVB) that reduces UVB-induced genotoxicity. DOI: 10.1111/php.13507.}
\end{cvitems}
}

\vspace{-0cm}

\cventry
{Research Assistant - Department of Biology - Advisor: Leopoldo Arrieta, PhD}
{Pedagogical and Technological University of Colombia}
{Tunja, Colombia}
{04/2016--03/2017}
{}
\vspace{-0.8cm}
\cventry
{}
{}
{}
{}
{
\begin{cvitems} % Description(s) of tasks/responsibilities
\item {Isolated and characterized \textit{Bacillus thuringiensis} from agricultural soils in BoyacÃ¡, Colombia}
\item {Evaluated the gene expression of cry1, the gene that encodes for the CryA endotoxin of \textit{B. thuringiensis}.}
\item {Obtained CryA endotoxin from \textit{B. thuringiensis} using solid-phase extraction and evaluating its insecticidal activity.}
\item {\textbf{Discovery}: proteolytic processing of the CryA pre-protein causes morphological and physiological changes in the midgut of the thrips' larvae.}
\end{cvitems}
}

\vspace{0.5cm}
  
%\cventry
%{Postdoctoral Associate - Department of Bacteriology - Advisor: Cameron R Currie, PhD}
%{University of Wisconsin-Madison}
%{Madison, WI}
%{04/2019--05/2019}
%{}
%\vspace{-0.4cm}
%\cventry
%{PhD Candidate - Department of Genetics - Advisor: Cameron R Currie, PhD}
%{}
%{}
%{08/2015--04/2019}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Built genomics-driven computational and analytic pipelines to uncover novel therapeutics and study the evolution of biosynthesis in free-living and host-associated microbes.}
%\item {Systems investigated include insects (leaf-cutting ants, honey bees, \& insects broadly), marine invertebrates, soil communities, \& the human microbiome.}
%\item {Led computational arm of Wisconsin Antimicrobial Drug Discovery \& Development Center (NIH CETR).}
%\end{cvitems}
%}

%\vspace{-0.4cm}
      
%\cventry
%{Lead Computational Biologist - Departments of Biology and Planetary Science - Advisor: Sarah S Johnson, PhD}
%{Harvard \& Georgetown Universities}
%{Cambridge, MA}
%{10/2013--10/2015}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Performed whole genome sequencing and metagenomic analysis of environmental samples from sulfur-rich, extreme environments with implications in microbial ecology, biogeochemistry, and exobiology.}
%\item {Characterized biosynthetic potential of metagenomic data.}
%\end{cvitems} 
%}

%\vspace{-0.4cm}

%\cventry
%{Head of Experimental Genomics}
%{Warp Drive Bio}
%{Cambridge, MA}
%{04/2013--08/2015}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Executed genomic-directed natural products drug discovery, high throughput Next Generation Sequencing (htNGS), computational biology, and molecular biology of actinomycetes and fungi.}
%\item {Designed and implemented genomic natural products searches over various scaffolds of business development and internal interest.}
%\item {Developed and curated computational pipelines and databases for assembly, annotation, and custom analysis of public and internal htNGS data (160,000 bacterial genomes, >150 closed and complete genomes) for analysis of novel polyketide, non-ribosomal peptide, and other natural product classes.}
%\item {Handled processing and management of sequence data, predictions, and analyses supporting multiple projects across discovery, molecular biology, engineering, and synthetic biology.}
%\item {Executed elucidation and prediction of novel chemical products of bacterial biosynthetic gene clusters and metabolic pathways (e.g. beta-lactams, aminoglycosides, rapamycin analogues, etc.).}
%\item {Developed internal pipelines for applied phylogenomic annotations and prioritizations of multiple data types to inform discovery and engineering efforts.}
%\item {Oversaw all lab and experimental support of actinomycete and fungal sequencing efforts for Illumina, Pacific Biosciences, and Oxford-Nanopore platforms.}
%\item {Bioinformatics software development to support molecular and synthetic biology efforts.}
%\item {Direct written and verbal communication of findings to senior leadership and business partners.}
%\item {Database management and delivery of sequence information to molecular biology, microbiology, and chemistry groups to aid drug discovery, strain engineering, and generation of expression constructs.}
%\end{cvitems}
%}

%\vspace{-0.4cm}

%\cventry
%{Research Assistant, Microbiology \& Computational Biology - Advisor: Tom\'{a}s Maira-Litr\'{a}n, PharmD, PhD}
%{Brigham \& Women's Hospital}
%{Boston, MA}
%{03/2013--08/2015}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Investigated \textit{in vivo} fitness, horizontal gene transmission, and pathogenesis of \textit{Acinetobacter baumannii}, \textit{Staphylococcus aureus}, \textit{Salmonella typhii}, and other virulent pathogens through microbiology, computational, and genomic techniques.}
%\item {Developed and optimized genetic tools to enable novel examinations of pathogen fitness, invasion, and virulence using high-throughput transposon-directed insertion site sequencing of infections in murine models.}
%\end{cvitems}
%}

%\vspace{-0.4cm}
       
%\cventry
%{Research Associate II, Molecular Biology Process Development}
%{Broad Institute of MIT \& Harvard}
%{Cambridge, MA}
%{01/2011--03/2013}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Independently designed development initiatives including supporting htNGS, microfluidics, and automation goals.}
%\item {Oversaw production and up-scaling of microbial mate-pair library construction (LC), integrated internal development with vendor technologies, and managed sample-tracking via real-time messaging to internal LIMS.}
%\item {Increased throughput of microbial LC Platform 4-fold by automation and protocol development.}
%\item{Worked extensively with mate-pair NGS LC, sequence analysis tools, genomic databases, statistical software, and programming/operating lab robotics.}
%\end{cvitems}
%}

%\vspace{-0.4cm}

%\cventry
%{Research Associate, Molecular Genetics - Advisor: Eric Rutledge, PhD}
%{Rensselaer Polytechnic Institute}
%{Troy, NY}
%{05/2010--12/2010}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Designed and developed protocols and operating procedures for transgenic \textit{Caenorhabditis elegans} cultures to model stress-induced neural degeneration and Parkinson's Disease.}
%\end{cvitems}
%}

%\vspace{-0.4cm}
       
%\cventry
%{Research Assistant, Microbiology}
%{BCR Biotech}
%{Jamestown, RI}
%{09/2009--12/2009}
%{
%\begin{cvitems} % Description(s) of tasks/responsibilities
%\item {Wrote and optimized protocols and methods for engineering synthetic biosensing functions in \textit{Bacillus} spores.}
%\end{cvitems}
%}

\end{cventries}

\vspace{-7mm}